### DISCUSSION TITLE
Unable to find declaration file for module 'redis'.

### DISCUSSION THREAD

**Discussion by sergiohdljr** [NodeRedis/node_redis#2698] [Q&A]
I just follow the install guide by the doc and when try to make any import receive the error : "Unable to find declaration file for module 'redis'."

try to install the @types/redis but the package is deprecated because redis provides its own type definitions.

Version installed : "3.1.2"

**Comment by tobias-hay** [comment]


**Reply by sergiohdljr** [reply]
not really, i just switch to ioredis https://www.npmjs.com/package/ioredis

**Reply by tobias-hay** [reply]
Ah yea, I realized that 3.1.2 was an old version and just bumped it to 4 and that problem no longer happens.
